http/src/api_error.rs

```diff
@@ -146,6 +146,8 @@ pub enum ErrorCode {
     MaximumNonGuildBansReached,
     /// Maximum number of bans fetches has been reached
     MaximumGuildBansFetchesReached,
+    /// Maximum number of uncompleted guild scheduled events reached (100)
+    MaximumNumberOfUncompletedGuildScheduledEventsReached,
```

Comment:
We usually name the variants themselves some sort of pared-down version of the comment. This one could be called `MaximumUncompletedEventsReached`, for example. We remove enough words from the comment so that it is still sort of clear what the error is for, but without having too long of a variant name.